testnets/elystestnet/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "elystestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Elys Network",
+  "chain_id": "elystestnet-1",
+  "bech32_prefix": "elys",
+  "daemon_name": "elysd",
+  "node_home": "$HOME/.elys",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uelys"
+      }
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://github.com/elys-network/elys",
+    "recommended_version": "v0.4.0",
+    "compatible_versions": [
+      "v0.4.0"
+    ],
+    "cosmos_sdk_version": "v0.46",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.34"
+    },
+    "ibc_go_version": "v6.1.0",
+    "cosmwasm_enabled": false,
+    "genesis": {
+      "genesis_url": "https://snapshots.polkachu.com/testnet-genesis/elys/genesis.json"
+    },
+    "versions": [
+      {
```

Comment:
can we please rearrange versions from earliest to latest? (i.e, start with v0.2.3, then v0.3.1, then v0.4.0)

Reply:
@JeremyParish69 solved